packages/compass-connection-import-export/src/components/import-modal.tsx

```diff
@@ -82,21 +86,22 @@ export function ImportConnectionsModal({
         displayName: (
           <>
             {conn.name}
-            {conn.isExistingFavorite && (
+            {conn.isExistingConnection && (
```

Comment:
What happens here in single connections world? Is this only true for favourites?

Reply:
This is just a rename to isExistingConnection (to not confuse anyhow that we're only working with favorites) but how we determine an existing connection is defined here - https://github.com/mongodb-js/compass/pull/5950/files#diff-ec1f998fbad63a79b9ef45caea37663b06897235f2af291a7612b8eea2b30744R103-R117

This changes between single and multi connection world.